torch/csrc/jit/mobile/nnc/context.cpp

```diff
@@ -48,17 +48,43 @@ OutputSpec::OutputSpec(const c10::IValue& value) {
   auto dict = value.toGenericDict();
   sizes_ = dict.at("sizes").toIntVector();
   dtype_ = dict.at("dtype").toScalarType();
+  if (dict.contains("qscale")) {
+    qscale_ = dict.at("qscale").toDouble();
+  }
+  if (dict.contains("qzero")) {
+    qzero_ = dict.at("qzero").toInt();
+  }
 }
 
 c10::IValue OutputSpec::serialize() const {
   c10::Dict<c10::IValue, c10::IValue> dict(
       at::StringType::get(), at::AnyType::get());
   dict.insert("sizes", sizes_);
   dict.insert("dtype", dtype_);
+  if (qscale_) {
```

Comment:
So looks like this won't break the backward compatibility for already compiled models.

cc: @priyaramani - as we discussed, I think we need to setup CIs to test already compiled models that we plan to run in production.